src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/ReportController.kt

```diff
@@ -41,4 +43,13 @@ class ReportController(context: ActionContext,
         tagRepository.updateTags(reportName, versionId, ReportVersionTags(versionTags, reportTags, listOf()))
         return okayResponse()
     }
+
+    fun getReportDrafts(reports: List<ReportWithPublishStatus>, drafts: List<ReportDraft>): List<ReportWithDraftsViewModel>
+    {
+        //val reports = reportRepository.getReportsWithPublishStatus()
+       // val drafts = reportRepository.getDrafts()
+        return reports.map { report ->
```

Comment:
WIll the ordering of these models be defined in the repo? What should it be? Order by date of most recent draft? (Don't want stale never-published drafts floating to the top because their report is first alphabetically or something.)

Reply:
Exactly, order by most recent draft.

Reply:
Do you think this makes sense, to fetch the reports with publish status. and then the drafts in 2 separate db calls? Or can you think of another way of organising things?